src/util/language_file.cpp

```diff
@@ -11,6 +11,10 @@ Author: Daniel Kroening, kroening@kroening.com
 #include "language.h"
 #include "language_file.h"
 
+// Defined here so that unique_ptr works properly
```

Comment:
Ah, of course. Would including language.h in language_file.h be a solution that works around the comments that you've now nicely added? They are very useful, but might be in violation of keep-it-simple. Just asking, completely up to you.